test/fragmenthandler.spec.js

```diff
@@ -27,72 +27,72 @@ describe('A FragmentHandler Component', function () {
     expect(fragmentHandler).to.exist;
   });
 
-  describe('gets the id of a collapsed section containing a fragment with the id of the hash',
+  describe('gets the id of a collapsible section containing a fragment with the id of the hash',
   function () {
 
-    it('possesses the method getIdOfCollapsedSection', function () {
-      expect(fragmentHandler.getIdOfCollapsedSection).to.be.a('function');
+    it('possesses the method getIdOfCollapsibleSection', function () {
+      expect(fragmentHandler.getIdOfCollapsibleSection).to.be.a('function');
     });
 
-    describe('the getIdOfCollapsedSection method', function () {
+    describe('the getIdOfCollapsibleSection method', function () {
 
-      it('returns null if there are no collapsed article sections in the document', function () {
+      it('returns null if there are no collapsible article sections in the document', function () {
 
         let docMock = {
           querySelectorAll: function (selector) {
-            if (selector === '.article-section--collapsed') {
+            if (selector === '.article-section--js') {
               return [];
             }
           },
           getElementById: function () {}
         };
 
-        expect(fragmentHandler.getIdOfCollapsedSection('arbitraryElementId',
+        expect(fragmentHandler.getIdOfCollapsibleSection('arbitraryElementId',
                                                        docMock)).to.be.null;
       });
 
-      it('returns null if hash does not identify a fragment of or within any collapsed sections',
+      it('returns null if hash does not identify a fragment of or within any collapsible sections',
       function () {
 
-        let collapsedSectionIds = [ 'collapsedA', 'collapsedB', 'collapsedC' ];
```

Comment:
How many collapsed sections are we going to have. If there is more than three do we need to think of a better naming convention?

Reply:
> How many collapsed sections are we going to have. If there is more than three do we need to think of a better naming convention?

This is a test fixture.

Reply:
Ok. I need to just test it locally.

Reply:
Just tested it locally. Looks ok. Anything you need me to test in particular to make 100% sure?